src/main/java/org/umcn/me/output/Save.java

```diff
@@ -0,0 +1,275 @@
+package org.umcn.me.output;
+
+import org.apache.log4j.Logger;
+import org.umcn.me.output.vcf.MobsterParser;
+import org.umcn.me.output.vcf.MobsterRecord;
+import org.umcn.me.output.vcf.MobsterRecordVCFWrapper;
+import org.umcn.me.output.vcf.MobsterToVCF;
+import org.umcn.me.pairedend.GRIPFunctions;
+import org.umcn.me.pairedend.MobilePrediction;
+import org.umcn.me.pairedend.Mobster;
+import org.umcn.me.util.CollectionUtil;
+import org.umcn.me.util.MobileDefinitions;
+import org.umcn.me.util.ReadName;
+
+import java.io.*;
+import java.util.*;
+
+import static org.umcn.me.output.Annotation.*;
+import static org.umcn.me.output.Filter.*;
+
+public class Save {
```

Comment:
Please rename class. This is quite ambiguous, and also bit trickly with keywords. What is the class "Saving"